bluebottle/members/serializers.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = BB_USER_MODEL
-        fields = ('id', 'first_name', 'last_name', 'username',
```

Comment:
That is already fixed. 

I do not really understand why this is part of the pull request. I already merged these commits to the release branch...